.vscode/spellright.dict

```diff
@@ -406,3 +406,8 @@ CAS
 OOP
 alignee
 UML
+UDRAL
+multirotors
+DRone
```

Comment:
typo. Drone

Reply:
This is intentional as part of "UAVCAN **DR**one Application Layer (UDRAL)".